petastorm/spark/spark_dataset_converter.py

```diff
@@ -28,6 +29,21 @@
 DEFAULT_ROW_GROUP_SIZE_BYTES = 32 * 1024 * 1024
 
 
+def _is_delta_available():
+    try:
+        # pylint: disable=import-error, unused-import, unused-variable
+        from delta.tables import DeltaTable  # noqa: F401
+        # pylint: enable=import-error, unused-import, unused-variable
+        return True
+    except ImportError:
+        return False
+
+
+def enable_delta():
```

Comment:
This method should be moved to test code.

Reply:
I put it here so that the user can use it to enable delta for their code simply by adding one line. I put the instruction in the warning message. Shall we keep it?

Reply:
I think it is ok.